docx/oxml/text/run.py

```diff
@@ -60,6 +61,19 @@ def clear_content(self):
         for child in content_child_elms:
             self.remove(child)
 
+    @property
+    def footnote_reference_ids(self) -> (list[int]|None):
+        """
+        Return all footnote reference ids (``<w:footnoteReference>``), or |None| if not present.
+        """
+        references = []
+        for child in self:
+            if child.tag == qn('w:footnoteReference'):
+               references.append(child.id)
+        if references == []:
+            references = None
```

Comment:
Would it make sense to return an empty list if there are no footnotes? I can see the benefit that users don't have to deal with None but instead check if the list is empty. What are your thoughts on this?